docs/src/references/agent/operation-workflow.md

```diff
@@ -328,6 +332,67 @@ Note that:
 - No `on_exit` nor `on_kill` status can be provided, as the script is not monitored.
 - If the script cannot be launched, the workflow will be moved to the final `"failed"` state.
 
+### Sub-Command Execution
+
+An operation workflow can trigger a command defined by another workflow.
+
+```toml
+["<state-name>"]
+command = "<operation-name>"
```

Comment:
> question: are we sure we don't want to explicitly state the filename of the sub-operation we're using?

This can be discussed. However, using the operation name has a main benefit. This can be used to trigger an operation with no user-defined workflow (e.g. a restart).

> As the top-level `operation = "..."` key inside the operation `.toml` file can have other name than the filename, it can be a bit annoying to find the sub-operation, because you have to search file contents.

And in practice, I fooled myself with such errors.

> But there's no concern of collisions, because multiple workflow files with the same name inside should cause the `tedge-agent` to quit with an error, is this right?

Currently, the agent doesn't quit on such errors but rejects with a warning such collisions.